core/web-assets/src/main/assets/js/apps/onms-requisitions/lib/scripts/model/RequisitionNode.js

```diff
@@ -4,6 +4,7 @@
 */
 
 const RequisitionInterface = require('./RequisitionInterface');
+const _ = require('lodash');
```

Comment:
@RangerRick added a comment recently to ensure that we really need this external dependency. If not, we should not introduce it.

Reply:
Fixed